InstantPrintPlugin.py

```diff
@@ -45,7 +45,7 @@ def initGui(self):
 
         self.toolButton.toggled.connect(self.__enableTool)
         self.iface.mapCanvas().mapToolSet.connect(self.__onToolSet)
-
+        
```

Comment:
Whitespace change (decent editors strip trailing whitespaces when saving the file, perhaps eric can be configured to do that)